webmails/snappymail/defaults/application.ini

```diff
@@ -20,6 +20,7 @@ allow_sync = On
 
 [defaults]
 contacts_autosave = On
+autologout = {{ ((PERMANENT_SESSION_LIFETIME | int)/60) | int }}
```

Comment:
`PERMANENT_SESSION_LIFETIME` with default value 10800 is not set by default in mailu.env. Either you must provide a default via applications.ini or set this default in applications.ini in case the variable is not set in mailu.env.

Reply:
Thanks for the hint, added a `default` filter.